api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -24,6 +24,17 @@ def add_item
             end
           end
 
+          def empty
+            if spree_current_order.nil?
+              render json: { error: "Order doesn't exist" }, status: 404
```

Comment:
@WitekL let's just raise `ActiveRecord::RecordNotFound` here, ok?

Reply:
We should also start utilising `i18n` 🙄